Makefile

```diff
@@ -4,38 +4,37 @@ BUILD_PATH := $(shell pwd)/.gobuild
 GS_PATH := $(BUILD_PATH)/src/github.com/giantswarm
 GOPATH := $(BUILD_PATH)
 
-BIN := $(PROJECT)
+BIN := $(PROJECT)ctl
 
 VERSION := $(shell cat VERSION)
 COMMIT := $(shell git rev-parse --short HEAD)
 
-.PHONY: all clean test ci-test
+.PHONY: all clean test ci-test deps
 
 SOURCE=$(shell find . -name '*.go')
 
-BUILD_COMMAND=go build -a -o $(BIN)
+BUILD_COMMAND=go build -o formicactl/$(BIN) github.com/giantswarm/formica/$(BIN)
 TEST_COMMAND=go test ./... -cover
 
-all: $(BIN)
+all: formica/$(BIN)
 
 clean:
-	rm -rf $(BUILD_PATH) $(BIN)
+	rm -rf $(BUILD_PATH) formicactl/$(BIN)
 
 .gobuild:
 	@mkdir -p $(GS_PATH)
 	@rm -f $(GS_PATH)/$(PROJECT) && cd "$(GS_PATH)" && ln -s ../../../.. $(PROJECT)
 	#
-	# Fetch private packages first (so `go get` skips them later)
-	# @GOPATH=$(GOPATH) builder go get github.com/spf13/cobra
-	# Pin versions of certain libs
-	@builder get dep -b v0.10.2 https://github.com/coreos/fleet.git $(GOPATH)/src/github.com/coreos/fleet
-	#
-	@GOPATH=$(GOPATH) builder go get github.com/juju/errgo
-	#
-	# Fetch public dependencies via `go get`
-	GOPATH=$(GOPATH) go get -d -v github.com/giantswarm/$(PROJECT)
+	# Fetch and pin packages
+	@builder get dep -b 76516ab4ae194e37aaae9c1f2fa5090553e541f3 https://github.com/coreos/fleet.git $(GOPATH)/src/github.com/coreos/fleet
+	@builder get dep -b 08cceb5d0b5331634b9826762a8fd53b29b86ad8 https://github.com/juju/errgo.git $(GOPATH)/src/github.com/juju/errgo
+	@builder get dep -b 65a708cee0a4424f4e353d031ce440643e312f92 https://github.com/spf13/cobra.git $(GOPATH)/src/github.com/spf13/cobra
+	@builder get dep -b 7f60f83a2c81bc3c3c0d5297f61ddfa68da9d3b7 https://github.com/spf13/pflag.git $(GOPATH)/src/github.com/spf13/pflag
+
+deps:
+	@${MAKE} -B -s .gobuild
 
-$(BIN): $(SOURCE) VERSION .gobuild
+formica/$(BIN): $(SOURCE) VERSION .gobuild
```

Comment:
We have `formicactl` as the main build target, which gets built to the top level. I can submit a PR, if you like?